internal/exec/shell_utils.go

```diff
@@ -187,7 +188,6 @@ func execTerraformShellCommand(
 			}
 			shellCommand = bashPath
 		}
-		shellCommand = shellCommand + " -l"
```

Comment:
It should be a login shell.

Reply:
Since the profile might override PS1, we should maybe define a rcfile

bash -l --rcfile /tmp/atmos_bashrc.536272

Reply:
Hi @osterman ,
I have tried but it seems custom PS1 is not working with the login shell

Reply:
Ok, understood. I am not able to get it working either.